project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/model/ProjectItemLens.kt

```diff
@@ -0,0 +1,8 @@
+package org.plan.research.minimization.plugin.model
+
+/**
+ * Interface that specifies a way how the property tester focuses on selected files on that level
+ */
+interface ProjectItemLens {
+    suspend fun focusOn(items: List<IJDDItem>, currentContext: IJDDContext)
```

Comment:
I don't know (I didn't get it), but the class and function names are strange. It's more about project transformation/DDItems application than focusing on something. What do you think?

Reply:
IMO, the main usage is to reduce the project items to a list of selected ones => using an interface focus on the specific part of the project. So it looks like a lens (not the Functional Programming one, yea)